html/css/styles.css

```diff
@@ -802,6 +802,15 @@ p.vspace { padding-top: 3px; padding-bottom: 3px; }
 
 .nofloat { clear: left; }
 
+.device_group {
+  color: #00a000;
```

Comment:
Why are you hard-defining colors here? You should probably use text-success class and <strong></strong> tags.

Reply:
> Why are you hard-defining colors here? You should probably use text-success class and strong tags.

You mean class fa-col-success?

Reply:
why coloring a link ? i wouldn't do it so color and style is defined on used style